<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(248, 236, 240);</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>GHI CHÚ</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_tongtien">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>NHÂN VIÊN XỬ LÝ</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_nvxl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
          background-color: white;
          border: 3px solid #3498db;  /* Viền xanh dương */
          border-radius: 10px;
          padding: 5px;
      }
      QComboBox::drop-down {
          border: none;
          background: transparent;
      }
      QComboBox::down-arrow {
          image: url(down-arrow.png); /* Thay bằng icon của bạn */
          width: 14px;
          height: 14px;
      }</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_luu">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(255, 255, 215); color: rgb(149, 106, 214); border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>LƯU</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>22</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(149, 106, 214);
font: 87 22pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>CẬP NHẬT ĐƠN HÀNG</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>TRẠNG THÁI</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_thanhtoan">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
          background-color: white;
          border: 3px solid #3498db;  /* Viền xanh dương */
          border-radius: 10px;
          padding: 5px;
      }
      QComboBox::drop-down {
          border: none;
          background: transparent;
      }
      QComboBox::down-arrow {
          image: url(down-arrow.png); /* Thay bằng icon của bạn */
          width: 14px;
          height: 14px;
      }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>THANH TOÁN</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_trangthai">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
          background-color: white;
          border: 3px solid #3498db;  /* Viền xanh dương */
          border-radius: 10px;
          padding: 5px;
      }
      QComboBox::drop-down {
          border: none;
          background: transparent;
      }
      QComboBox::down-arrow {
          image: url(down-arrow.png); /* Thay bằng icon của bạn */
          width: 14px;
          height: 14px;
      }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>TỔNG TIỀN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>MÃ KHÁCH HÀNG</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_maKH">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_note">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_maDH">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>MÃ ĐƠN HÀNG</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
